VEDA_OS01/models.py

```diff
@@ -4,13 +4,21 @@
 import json
 import uuid
 from django.db import models
+from django.utils import timezone
 from fernet_fields import EncryptedTextField
 from model_utils.models import TimeStampedModel
 
 from opaque_keys import InvalidKeyError
 from opaque_keys.edx.keys import CourseKey
 
 
+def current_year():
+    """
+    Returns the current year.
+    """
+    return timezone.now().strftime('%Y')
```

Comment:
How about saving the whole datatime instead of year. Also, be aware of .now() VS .now when we're using it as fields in django models.

https://stackoverflow.com/questions/29733203/django-default-datetime-now-in-models-always-saves-same-datetime-after-uwsgi-r

Reply:
it is callable.

Reply:
How about it being Date field instead?

Reply:
Yeah that's what i said in first line.